stable/heapster/templates/deployment.yaml

```diff
@@ -30,6 +30,10 @@ spec:
           timeoutSeconds: 5
         resources:
 {{ toYaml .Values.resources | indent 12 }}
+{{- if .Values.env }}
```

Comment:
Nit: the `if` and `end` statements should be indented just one level less than the block they wrap.